frontend/Ingredishare/app/src/main/java/com/example/andriod/ingredishare/MainActivity.java

```diff
@@ -75,35 +65,6 @@
     private GlobalRequestQueue mReqQueue;
     private RequestQueue requestQueue;
 
-
-    /*
-    Create a getRequestQueue() method to return the instance of
-    RequestQueue.This kind of implementation ensures that
-    the variable is instatiated only once and the same
-    instance is used throughout the application
-    */
-    public RequestQueue getRequestQueue() {
```

Comment:
these methods in `MainActivity.java` aren't being used anymore, so I took them out.